lib/Doctrine/DBAL/Migrations/Tools/Console/Command/ExecuteCommand.php

```diff
@@ -86,7 +86,13 @@ public function execute(InputInterface $input, OutputInterface $output)
             $version->writeSqlFile($path, $direction);
         } else {
             if ($input->isInteractive()) {
-                $execute = $this->getHelper('dialog')->askConfirmation($output, '<question>WARNING! You are about to execute a database migration that could result in schema changes and data lost. Are you sure you wish to continue? (y/n)</question>', false);
+                $questionText = 'WARNING! You are about to execute a database migration that could result in schema changes and data lost. Are you sure you wish to continue? (y/n)';
+                if ($this->getHelperSet()->has('question')) {
+                    $question = new \Symfony\Component\Console\Question\ConfirmationQuestion($questionText);
```

Comment:
I didn't because this would break symfony 2.3 (QuestionHelper is not available there) based on your comment here: https://github.com/doctrine/migrations/pull/198#issuecomment-99458078